src/uct/ib/dc/dc_mlx5.h

```diff
@@ -62,6 +62,11 @@ struct ibv_ravh {
         (_self)->flags |= UCT_DC_MLX5_IFACE_FLAG_##_flag_name##_FULL_HANDSHAKE; \
     }
 
+#define UCT_DC_MLX5_DECL_EMPTY_DCI(_empty_dci) \
+    uct_dc_dci_t _empty_dci      = {{{0}}}; \
```

Comment:
this macro has both var declaration and assign statements so it must always be placed after all var declarations
do we really have to set all its fields to 0?
maybe define a function uct_dc_mlx5_init_empty_dci that sets pool_index and qp_num only?